e2e-pw/src/oss/poms/panels/embeddings-panel.ts

```diff
@@ -0,0 +1,75 @@
+import { Locator, Page, expect } from "src/oss/fixtures";
+import { EventUtils } from "src/shared/event-utils";
+import { SelectorPom } from "../selector";
+import { PanelPom } from "./panel";
+
+export class EmbeddingsPom {
+  readonly locator: Locator;
+  readonly plotContainer: Locator;
+  readonly selector: SelectorPom;
+  readonly colorbySelector: SelectorPom;
+  readonly asserter: EmebddingsAsserter;
+  readonly pp: Locator;
+  readonly lassoTool: Locator;
+
+  constructor(
+    private readonly page: Page,
+    private readonly eutils: EventUtils
+  ) {
+    this.pp = page;
```

Comment:
I think this can be removed, and `page: Page` can be `readonly` instead of `private readonly`